devmaster/modules/devshop/devshop_dothooks/devshop_dothooks.module

```diff
@@ -149,6 +149,7 @@ function devshop_dothooks_run_hook($hook, $site, $task = null) {
   $env = $_SERVER;
   $env['DEVSHOP_PROJECT'] = $environment->project_name;
   $env['DEVSHOP_ENVIRONMENT'] = $environment->name;
+  $env['DEVSHOP_ENV_URI'] = $environment->uri; // the domain
```

Comment:
I try to be explicit and avoid abbreviations where possible.

Change to DEVSHOP_ENVIRONMENT_URI